### PR TITLE
LPS-146373 Adapt tests

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/components/layout-data-items/FragmentWithControls.test.js [hunk 1]
```diff
@@ -104,7 +104,13 @@ const renderFragment = ({
 };
 
 describe('FragmentWithControls', () => {
-	afterEach(cleanup);
+	afterEach(() => {
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/components/layout-data-items/FragmentWithControls.test.js:107]: Shouldn't this be "beforeEach"? :thinking:

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/_Sidebar.scss [hunk 2]
```diff
@@ -177,6 +178,16 @@ html#{$cadmin-selector} .page-editor__wrapper {
 	transition: padding-left ease $defaultTransitionDuration,
 		padding-right ease $defaultTransitionDuration;
 	will-change: padding-left, padding-right;
+	max-height: calc(
+		100vh -
+			(
+				var(--control-menu-height) + var(--toolbar-height) +
+					var(--change-tracking-indicator-height, 0px) +
+					$toolbarBorders
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/_Sidebar.scss:187]: you have to escape this (`#{$toolbarBorders}`)

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/components/layout-data-items/FragmentWithControls.test.js (hunk 1 above)
Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/components/layout-data-items/FragmentWithControls.test.js:107]: Unda dios, toda la razón

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
ci:reevaluate:1387401_2730

**Comment by p2kmgcl** [comment]
ci:forward

... YOU are unable :cry: 

> Unable to [reevaluate](https://test-1-35.liferay.com/job/test-portal-evaluate-pullrequest/41/) previous pull request result. Please wait for this [acceptance upstream build](https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/3634) to complete, and attempt reevaluation again with the CI command:
> 
> ci:reevaluate:1387401_2730

**Comment by ealonso** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
ci:forward

Mira chiquillo, follow the PR

**Comment by victorg1991** [comment]
resent here https://github.com/liferay-echo/liferay-portal/pull/9201

**Pull request closed without merging**